apps/backend/test/mocks/classes/transaction_mock.ts

```diff
@@ -1,43 +1,33 @@
 import { faker } from '@faker-js/faker';
 import { format } from 'date-fns';
-import { Payment } from './payment_mock';
-import { BaseData } from '../types/interface';
+import { PaymentMock } from './payment_mock';
+import { PaymentEntity } from './../../../src/transaction/entities/payment.entity';
```

Comment:
nitpick: for consistency, we can remove `./`. TBH I like it better with, but yeah

Reply:
> Entity specs don't strike me as particularly useful, but I'd love to hear your take on their value. If it's just coverage, I'd argue for removing

It's 90% for coverage, and also for examples in the codebase/regression testing. I thought it would be unlikely for these to change (and if they do change we'd need to make sure the fields used in reconciliation aren't affected or altered) but I'm sure the tests for reconciliation are enough to cover this, I'm happy to remove if you would prefer.